.github/workflows/trufflehog.yml

```
@@ -0,0 +1,40 @@
+name: "TruffleHog"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+  
+  schedule:
+    - cron: "0 0 * * *" # Once a day
```

Wouldn't it make more sense to perform the secret scanning on each pull request, too? After merging it would be too late. I copied this part from the [codeql workflow](https://github.com/eclipse-tractusx/puris/blob/main/.github/workflows/codeql.yml#L32C1-L40C21)

```suggestion
on:
  push:
    branches: ["main"]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: ["main"]
  schedule:
    - cron: "36 1 * * 0"
  workflow_dispatch:
```